examples/playground/src/deck-with-gmaps.js

```diff
@@ -0,0 +1,109 @@
+/* global console, document, window */
+import React, {Component} from 'react';
+import {GoogleMapsOverlay} from '@deck.gl/google-maps';
+
+const HOST = 'https://maps.googleapis.com/maps/api/js';
+const LOADING_GIF = 'https://upload.wikimedia.org/wikipedia/commons/d/de/Ajax-loader.gif';
+
+const style = {
+  height: 1000,
+  width: 1000
+};
+
+function injectScript(src) {
+  return new Promise((resolve, reject) => {
+    const script = document.createElement('script');
+    script.src = src;
+    script.addEventListener('load', resolve);
+    script.addEventListener('error', e => reject(e.error));
+    document.head.appendChild(script);
+  });
+}
+
+function deepEqual(x, y) {
+  const ok = Object.keys;
+  const tx = typeof x;
+  const ty = typeof y;
+  return x && y && tx === 'object' && tx === ty
+    ? ok(x).length === ok(y).length && ok(x).every(key => deepEqual(x[key], y[key]))
+    : x === y;
+}
+
+function loadGoogleMapApi(apiKey, onComplete) {
```

Comment:
Oh, this isn't already done by `deck.gl/google-maps`? 

Also, call it GoogleMaps instead of GoogleMap for consistency?